app/main.py

```diff
@@ -1,6 +1,24 @@
-# write your imports here
+from __future__ import annotations
 
+from app.cinema.bar import CinemaBar
+from app.people.customer import Customer
+from app.cinema.hall import CinemaHall
+from app.people.cinema_staff import Cleaner
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+
+def cinema_visit(
+        customers: list[Customer],
```

Comment:
sorry, if you create here instances you get dicts
```suggestion
        customers: list[dict],
```